src/main/java/taxi/model/Driver.java

```diff
@@ -39,6 +41,22 @@ public void setLicenseNumber(String licenseNumber) {
         this.licenseNumber = licenseNumber;
     }
 
+    public String getLogin() {
+        return login;
+    }
+
+    public void setLogin(String login) {
+        this.login = login;
+    }
+
+    public String getPassword() {
+        return password;
+    }
+
+    public void setPassword(String password) {
+        this.password = password;
+    }
+
```

Comment:
Seems like you haven't override equals() and hashCode() with new fields login and password

Reply:
Fixed 😬